### PR TITLE
Backport Quickstart changes

### PR DESCRIPTION
**Pull request by s0enke**
This backports changes from the [Quickstarts Repo](https://github.com/aws-quickstart/quickstart-superwerker). Hopefully, this doesn't happen too often since usually, we push our changes/features to the Quickstart.

### CODE REVIEW

File: templates/superwerker.template.yaml [hunk 1]
```diff
@@ -31,42 +31,41 @@ Metadata:
...
+        default: Quick Start S3 bucket Region
+      QSS3KeyPrefix:
+        default: Quick Start S3 key prefix
+      Subdomain:
+        default: Subdomain for automated DNS configuration
 Parameters:
   Domain:
     Type: String
-    Description: Domain used for RootMail feature
+    Description: Domain used for root mail feature
```
Comment (sbstjn) [inline templates/superwerker.template.yaml:57]: `RootMail` is the product name. I would not change this.

**Comment by sbstjn** [comment]
Shall we delete the `docs/partner_editable` folder to get rid of the continuous backporting?

File: templates/superwerker.template.yaml (hunk 1 above)
Comment (s0enke) [reply templates/superwerker.template.yaml:57]: Yes, TBH I did not look at the changes in detail. My primary goal was to gain parity again. Would this be a blocker for merging this PR? We could also open a PR at the QuickStarts repo to name it back to `RootMail` over there so we decouple the two concerns.

**Comment by s0enke** [comment]
> Shall we delete the `docs/partner_editable` folder to get rid of the continuous backporting?

Would be OK for me. I could do this in a follow-up PR.

**Comment by sbstjn** [comment]
Merge it 🏋️‍♀️

**Pull request merged**